js/main.js

```diff
@@ -131,7 +131,7 @@ function renderEntries(entry) {
   $iconDiv.appendChild($editIcon);
   $textDiv.appendChild($entryText);
 
-  $entryRow.addEventListener('click', handleEdit);
+  $entriesList.addEventListener('click', handleEdit);
```

Comment:
This shouldn't be registered inside here. You're adding X listeners, but we want one. Add your event listener _outside_ of your entry-rendering function.